web/client-api/src/main/java/io/deephaven/web/client/api/JsLayoutHints.java

```diff
@@ -15,8 +15,7 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-@TsInterface
-@JsType(name = "LayoutHints", namespace = "dh")
+@TsName(namespace = "dh", name = "LayoutHints")
```

Comment:
This technically works, but when we effectively have an enum, we're preferring to move those to their own distinct type. So

```java
@JsTypeDef(jsType = "string")
@JsType(namespace = "dh")
public class SearchDisplayMode {
    public static final String SEARCH_DISPLAY_DEFAULT = "Default",
            SEARCH_DISPLAY_HIDE = "Hide",
            SEARCH_DISPLAY_SHOW = "Show";
}
```

Then these annotations can stay as-is, since it is more appropriate for this to be an interface, right?